--- cherokee-0.99.29/configure.in.orig	2009-11-16 17:42:47.000000000 +0100
+++ cherokee-0.99.29/configure.in	2009-11-28 02:11:19.000000000 +0100
@@ -1281,13 +1281,7 @@
 if test "x$PHPCGI" = "x"; then
    AC_MSG_RESULT([no])
 else
-   $PHPCGI -v -d session.save_path=/tmp | grep 'cgi-fcgi' >/dev/null 2>/dev/null
-   if test $? != 0; then
-      PHPCGI=""
-      AC_MSG_RESULT([no])
-   else
-      AC_MSG_RESULT([yes])
-   fi
+   AC_MSG_RESULT([yes])
 fi
 
 if test "x$PHPCGI" = "x"; then
