--- cherokee-0.9.4/cherokee.conf.sample.pre.orig	2008-09-26 23:29:36.000000000 +0200
+++ cherokee-0.9.4/cherokee.conf.sample.pre	2008-10-21 01:15:39.000000000 +0200
@@ -16,7 +16,7 @@
 server!keepalive = 1
 server!keepalive_max_requests = 500
 server!server_tokens = full
-server!panic_action = %prefix%/bin/cherokee-panic
+server!panic_action = %prefix%/sbin/cherokee-panic
 server!pid_file = %localstatedir%/run/cherokee.pid
 
 # Default virtual server
--- cherokee-1.0.10/admin/configured.py.pre.orig	2010-10-28 11:42:31.000000000 +0200
+++ cherokee-1.0.10/admin/configured.py.pre	2010-11-30 18:36:22.000000000 +0100
@@ -17,7 +17,7 @@
 CHEROKEE_ADMINDIR     = join (PREFIX, "share/cherokee/admin")
 CHEROKEE_ICONSDIR     = join (PREFIX, "share/cherokee/icons")
 CHEROKEE_THEMEDIR     = join (PREFIX, "share/cherokee/themes")
-CHEROKEE_PANIC_PATH   = join (PREFIX, "bin/cherokee-panic")
+CHEROKEE_PANIC_PATH   = join (PREFIX, "sbin/cherokee-panic")
 CHEROKEE_PLUGINDIR    = join (LIBDIR, "cherokee")
 CHEROKEE_DATADIR      = join (DATADIR, "cherokee")
 CHEROKEE_DEPSDIR      = join (DATADIR, "cherokee/deps")
